react/features/base/connection/actions.native.js

```diff
@@ -370,10 +370,7 @@ export function disconnect() {
         if (connection_) {
             promise = promise.then(() => connection_.disconnect());
         } else {
-            // FIXME: We have no connection! Fake a disconnect. Because of how the current disconnec is implemented
-            // (by doing the diconnect() in the Conference component unmount) we have lost the location URL already.
-            // Oh well, at least send the event.
-            promise.then(() => dispatch(_connectionDisconnected({}, '')));
+            logger.error('No connection found while disconnecting.');
```

Comment:
because the disconnect is dispatched unconditionally maybe it might be too much to log it on the error level ? Isn't that logged when the conference is joined for the first time ?

If I understand correctly this will also be logged every time the disconnect is called when the old conference is on the load config stage.

Reply:
True, I'll make it debug.